frontend/src/store/versionStore.ts

```diff
@@ -126,7 +126,9 @@ export const useVersionStore = defineStore('version', () => {
   function findVersionsByObjectId(objectId: string) {
     return state.versions.value.filter((x: Version) => x.objectId === objectId);
   }
-
+  function findS3VersionByObjectId(objectId: string) {
```

Comment:
this is fine. but for future reference, findObjectById() already returns the s3VersionId.
so i think you can do formData.value.s3VersionId